apps/web/src/components/nav/EnvironmentSelect/EnvironmentSelect.tsx

```diff
@@ -1,55 +1,52 @@
-import { Select, When } from '@novu/design-system';
+import { useState } from 'react';
+
+import { Select, IconConstruction, IconRocketLaunch } from '@novu/design-system';
+
 import { css } from '@novu/novui/css';
 import { navSelectStyles } from '../NavSelect.styles';
-import { EnvironmentPopover } from './EnvironmentPopover';
-import { useEnvironmentSelect } from './useEnvironmentSelect';
+import { useEnvironment } from '../../../components/providers/EnvironmentProvider';
+import { BaseEnvironmentEnum } from '../../../constants/BaseEnvironmentEnum';
 
-export const EnvironmentSelectRenderer: React.FC<ReturnType<typeof useEnvironmentSelect>> = ({
-  icon,
-  isPopoverOpened,
-  setIsPopoverOpened,
-  handlePopoverLinkClick,
-  readonly,
-  ...selectProps
-}) => {
-  return (
-    <EnvironmentPopover
```

Comment:
The popover about changes is not relevant in V2 anymore and was a bit buggy in V1. So, it was removed.